cmd/publishing-bot/publisher.go

```diff
@@ -137,7 +137,7 @@ func (p *PublisherMunger) construct() error {
 
 		// clone the destination repo
 		dstDir := filepath.Join(p.baseRepoPath, repoRule.DestinationRepository, "")
-		dstURL := fmt.Sprintf("https://github.com/%s/%s.git", p.config.TargetOrg, repoRule.DestinationRepository)
+		dstURL := fmt.Sprintf("https://%s/%s/%s.git", p.config.GithubHost, p.config.TargetOrg, repoRule.DestinationRepository)
```

Comment:
while at it: can we unify the way we build the url? We have the `"x" + "/" + y + "/"` style, the `filepath.Join` style and now this one. This one is probably the most correct and idiomatic.

Reply:
@sttts: I agree that we should rewrite `x + "/" + y + "/" +z` with `fmt.Sprintf` if there are more than three terms. 

`filepath.Join` is different though, it needs to handle `os.PathSeparator`.